src/pages/Graph/SummaryPanelEdge.tsx

```diff
@@ -347,17 +370,22 @@ export default class SummaryPanelEdge extends React.Component<SummaryPanelPropTy
     );
   };
 
+  private hasSupportedCharts = edge => {
+    const sourceData = nodeData(edge.source());
+    const destData = nodeData(edge.target());
+    const sourceMetricType = getNodeMetricType(sourceData);
+    const destMetricType = getNodeMetricType(destData);
+
+    // service-to-service edges are unsupported because they represent aggregations (of multiple workload to service edges)
+    const chartsSupported = sourceMetricType !== NodeMetricType.SERVICE || destMetricType !== NodeMetricType.SERVICE;
+    return chartsSupported;
+  };
+
   private hasHttpMetrics = (edge): boolean => {
-    if (edge.data('rate')) {
-      return true;
-    }
-    return false;
+    return edge.data('rate');
   };
 
   private hasTcpMetrics = (edge): boolean => {
-    if (edge.data('tcpSentRate')) {
-      return true;
-    }
-    return false;
+    return edge.data('tcpSentRate');
```

Comment:
This says the function returns a "boolean" but the return statement returns the data rate (presumably a number, not a boolean). Is this correct? Why was this change made, in other words? Why not:

`return edge.data('tcpSentRate') ? true : false;`

Reply:
uhm... fine, I'll change these.